community/cypher/acceptance/src/test/scala/org/neo4j/internal/cypher/acceptance/LoadCsvAcceptanceTest.scala

```diff
@@ -381,6 +381,20 @@ class LoadCsvAcceptanceTest
     result.toList should equal(List(Map("field" -> "something")))
   }
 
+  test("should restrict file urls to be rooted within an authorized directory") {
+    val dir = createTempDirectory("loadcsvroot")
+
+    val db = new TestGraphDatabaseFactory()
```

Comment:
Yeah you are right we already do that at a couple of places. Never mind we'll clean up at some other time. If you are interested I think you can add

```
override def databaseConfig() = super.databaseConfig() ++ Map(GraphDatabaseSettings.allow_file_urls -> "false", GraphDatabaseSettings.load_csv_file_url_root, dir.toString)
```

and then your test would only be 

```
intercept[LoadExternalResourceException] {
    execute(s"LOAD CSV FROM 'file:///../foo.csv' AS line RETURN line[0] AS field").size
}.getMessage should endWith(" file URL points outside configured import directory")
```